### ISSUE TITLE
hcana compile error

### ISSUE THREAD

**Issue by Yero1990** [JeffersonLab/hcana#486]
I update to the most recent version of podd/hcana and got the following error after re-compiling hcana with scons:

```sh
podd/hana_decode/libdc.so: undefined reference to `DAQconfig::parse(unsigned long)'
podd/Podd/libPodd.so: undefined reference to `DAQInfoExtra::GetFrom(TObject*)'
podd/Podd/libPodd.so: undefined reference to `DAQInfoExtra::AddTo(TObject*&, DAQInfoExtra*)'
podd/Podd/libPodd.so: undefined reference to `DAQInfoExtra::GetExtraInfo(TObject*)'
collect2: error: ld returned 1 exit status
scons: *** [hcana] Error 1
scons: building terminated because of errors.
```

I did noticed when updating podd there were multiple tags, but I assume that the users do not have to change to a specific one.

**Comment by hansenjo** [comment]
Sorry about that. There was a regression in the SCons build system. This should be fixed in commit 286763e3 on the develop branch.

About the tags: Try doing the following:
```
git checkout develop
git pull origin develop
git submodule update
cd podd
git tag -d Release-173
git fetch --tags
cd ..
git tag -d hcana-0.95
git fetch --tags
```
Let me know if this solves it for you.

**Comment by Yero1990** [comment]
OK, now hcana is working.

Thanks,
